core/src/test/kotlin/team/backend/support/Compression.kt

```diff
@@ -0,0 +1,31 @@
+package team.backend.support
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.comparables.shouldBeGreaterThan
+import io.kotest.matchers.shouldBe
+
+class Compression: StringSpec({
```

Comment:
I think it would be better to add the suffix 'Test' after the class name for test code.

Reply:
Good opinions !